extmod/modussl_mbedtls.c

```diff
@@ -74,6 +85,27 @@ STATIC void mbedtls_debug(void *ctx, int level, const char *file, int line, cons
 }
 #endif
 
+STATIC mp_obj_t mod_ssl_errstr(mp_obj_t err_in) {
+    size_t err = mp_obj_get_int(err_in);
+    vstr_t vstr;
+    vstr_init_len(&vstr, 80);
+
+    // Including mbedtls_strerror takes about 16KB on the esp32 due to all the strings
+#if 1
```

Comment:
Note to self: this needs adaptation per https://github.com/tve/micropython/commit/cdb014380cd00675f8c05abd768c0982e6a27330